src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -171,6 +173,10 @@ private function registerReceivers(ContainerBuilder $container)
             }
         }
 
+        if (1 === \count($taggedReceivers) && $container->hasDefinition('console.command.messenger_consume_messages')) {
+            $container->getDefinition('console.command.messenger_consume_messages')->setArgument(3, (string) current($receiverMapping));
```

Comment:
The `(string)` cast looks unnecessary to me?

We usually change the definition in xml files to define a placeholder for the argument with a comment and use `replaceArgument` instead of `setArgument`. To me, that's a good hint to known this argument is replaced by a pass or an extension when reading the sources.

Reply:
You are right, I've changed it to use `replaceArgument`. Though, I kept the `(string)` to make the cast from `Reference` to `string` explicit.